lib/components/dashboard-modules/SunnyTimeframe.tsx

```diff
@@ -1,24 +1,18 @@
 import { FC, useState } from 'react';
-
 import NumberDisplay from './NumberDisplay';
-
-import { getNextThresholdIndex } from 'lib/utils';
-
-import { graphThreshold } from 'lib/graphs';
-
-import { useSiteData } from 'lib/hooks';
-import { hoursToMinutes, millisecondsToHours } from 'date-fns';
-import useDateFormatter from '~/lib/hooks/useDateFormatter';
-
-const SunnyTimeframe: FC<{ siteUUID: string }> = ({ siteUUID }) => {
-  const { forecastData, isLoading, installed_capacity_kw } =
-    useSiteData(siteUUID);
+import { getNextThresholdIndex, graphThreshold } from 'lib/graphs';
+import dayjs from 'dayjs';
+import { useSiteData } from '~/lib/sites';
+import { Site } from '~/lib/types';
+import { timeFormat } from '~/lib/format';
+import { useSiteTime } from '~/lib/time';
+
+const SunnyTimeframe: FC<{ site: Site }> = ({ site }) => {
+  const { forecastData, isLoading } = useSiteData(site.site_uuid);
   const [isRelativeTime, setIsRelativeTime] = useState(false);
-  const { timeFormatter } = useDateFormatter(siteUUID);
+  const { timezone } = useSiteTime(site);
 
-  const thresholdCapacityKW = installed_capacity_kw
-    ? installed_capacity_kw * graphThreshold
-    : 0;
+  const thresholdCapacityKW = site.installed_capacity_kw * graphThreshold;
```

Comment:
i think it's fine to make this use multiple sites too

Reply:
I'll hold off on this PR til @rohanvan123 is done then, since he's making the changes to support multiple sites in this components.